Openai-codex-best-practices-2026.html

```
@@ -0,0 +1,2221 @@
+
+          <section id="loop">
+            <span class="step-tag">Core Concept</span>
+            <h2>2. Codexの基本動作ループを理解する</h2>
+            <p>
+              ベストプラクティスの前提として、Codexがどう動いているかを押さえておきましょう。プロンプトを送信すると、Codexは「モデルを呼び出す
+              →
+              出力が指示するアクション(ファイル読み書き・コマンド実行・ツール呼び出し)を実行する」というループを、タスクが完了するかユーザーがキャンセルするまで繰り返します。
+            </p>
+            <div class="diagram">
+              <div class="mermaid">
+                flowchart TD A[ユーザーがプロンプトを送信] --> B[Codexがモデルを呼び出す] B -->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)